templates/home/index.html

```diff
@@ -407,13 +407,23 @@ <h4>IFSC Code :</h4>
                 <div class="contact-details-div">
                      <div class="contact-details contact-details-left">
                         Name <br>
-                        98726 - 829xx <br>
-                        xyxysa@iiitdmj.ac.in
+<<<<<<< HEAD
+                        <a class="donateLink" href="tel:98726 - 829xx"> 98726 - 829xx </a><br>
+                        <a class="donateLink" href="mailto:xyxysa@iiitdmj.ac.in"> xyxysa@iiitdmj.ac.in </a>
+                     </div>
+                     <div class="contact-details contact-details-right">
+                        Name <br>
+                        <a class="donateLink" href="tel:98726 - 829xx"> 98726 - 829xx </a><br>
+                        <a class="donateLink" href="mailto:xyxysa@iiitdmj.ac.in"> xyxysa@iiitdmj.ac.in </a>
+=======
+                        <a style="color: #000;" href="tel:98726 - 829xx" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > 98726 - 829xx </a><br>
+                        <a style="color: #000;" href="mailto:xyxysa@iiitdmj.ac.in" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > xyxysa@iiitdmj.ac.in </a>
                      </div>
                      <div class="contact-details contact-details-right">
                         Name <br>
-                        98726 - 829xx <br>
-                        xyxysa@iiitdmj.ac.in
+                        <a style="color: #000;" href="tel:98726 - 829xx" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > 98726 - 829xx </a><br>
+                        <a style="color: #000;" href="mailto:xyxysa@iiitdmj.ac.in" onMouseOver="this.style.color='#ff8c00'"onMouseOut="this.style.color='#000'" > xyxysa@iiitdmj.ac.in </a>
+>>>>>>> 4ff1a8a09687423f1a1f1aeb256d3f35a9761c44
```

Comment:
It seems like you didn't resolve the merge conflicts properly. The code above and below the `======` line are two versions of the same code, and you need to decide which one to keep. You can watch some video on how to resolve merge conflicts on YouTube.